src/star-rating/index.js

```diff
@@ -0,0 +1,35 @@
+const simpleIcons = require("simple-icons");
+const starFilledSVG = require("./star-filled.svg.js");
+const starRegularSVG = require("./star-regular.svg.js");
+const starRatingComponentStyles = require("./index.style");
+
+const starRating = async (data = {}) => {
+  const { skill, rating } = data;
+  let content = ``;
+  Array.from({ length: 5 }).map((_r, index) => {
+    if (index < parseInt(rating)) {
+      content += starFilledSVG();
+    } else content += starRegularSVG();
+  });
```

Comment:
Im guessing this might be un necessary to have two separate methods here. Rather we can use one method and send color as param to get filled svgs

Reply:
I observed that both the SVGs have different `path` compositions so updating based on color was not giving expected result. Instead, I have combined SVGs for both stars in 1 file and added another param `isFilled` in the file which if sent `true` would return `path` for filled star and if not would return  `path` for regular star. Feel free if you have suggestions to add.

Reply:
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="1" stroke-linecap="round" stroke-linejoin="round" class="feather feather-star"><polygon points="12 2 15.09 8.26 22 9.27 17 14.14 18.18 21.02 12 17.77 5.82 21.02 7 14.14 2 9.27 8.91 8.26 12 2"></polygon></svg>

We can use this star with `stroke-width` 1 and pass in `fill` as param based on we need color or not. I took the icon from https://feathericons.com/?query=star

Reply:
done in [4bd3d4f](https://github.com/harish-sethuraman/readme-components/pull/12/commits/4bd3d4f1245cb61fd9bd564deeddf106069f9e4c)